strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/layout/maven/MavenArtifactController.java

```diff
@@ -51,8 +51,8 @@
 {
 
     @PreAuthorize("authenticated")
-    @RequestMapping(value = "greet", method = RequestMethod.GET)
-    public ResponseEntity greet()
+    @RequestMapping(value = "/{storageId}/{repositoryId}/", method = RequestMethod.GET)
```

Comment:
Is the backslash needed?

Reply:
i make request mapping same as NugetArtifactController's checkRepositoryAccess method